internal/pwru/types.go

```diff
@@ -34,6 +34,7 @@ type Flags struct {
 	FilterDstPort  uint16
 	FilterPort     uint16
 	FilterTrackSkb bool
+	FilterPcap     string
```

Comment:
I'd suggest to drop all previous filters, and allow users to specify the filter pcap in the same way as with tcpdump. For example, `pwru --output-tuple 'host 1.1.1.1 and port 80'`. Let's do it as a follow-up.